controllers/glanceapi_controller.go

```diff
@@ -125,9 +155,7 @@ func (r *GlanceAPIReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		instance.Status.Conditions.Init(&cl)
 
 		// Register overall status immediately to have an early feedback e.g. in the cli
-		if err := r.Status().Update(ctx, instance); err != nil {
-			return ctrl.Result{}, err
-		}
+		return ctrl.Result{}, err
```

Comment:
same as `return ctrl.Result{}, nil`, so could also return this explicitly.

Reply:
Changed